.gitignore

```diff
@@ -26,3 +26,4 @@
 # Ignore master key for decrypting credentials and more.
 /config/master.key
 /config/database.yml
+/vendor/*
```

Comment:
Sao lại bỏ vendor đi

Reply:
@luongvietdung file vendor em phải cache lại file gem để dùng được gem whenever nên e bỏ vendor vào trong .gitignore để tránh bị file đẩy lên nhiều quá ạ

Reply:
Tại sao phải cache lại file gem hả chú

Reply:
@luongvietdung Nó ghi là k tìm được file gem ạ. E có remove file gem và bundle lại rồi nhưng không được!
Nhưng nếu e bundle  --cache vào file vendor thì chạy ổn định và k có lỗi gì nhưng e thấy cách ấy k được tối ưu lắm. Vậy nên e xin A cho e hướng giải quyết tốt hơn ạ
![image](https://user-images.githubusercontent.com/43009261/61603813-4921b480-ac69-11e9-8505-e55cc98d7383.png)

Reply:
@DuyNguyen05 Nhưng cái lỗi này liên quan gì đến gem whenver đâu?
Chú chụp cả full lỗi anh xem nào

Reply:
@luongvietdung A ơi log n k hiện ra lỗi j cả ạ
Nó chỉ hiện trong log của cron thôi ạ

Reply:
@DuyNguyen05 Neu the chi cho cai thu muc cua chu vao vendor thoi. ko nen bo ca thu muc vendor